src/tmpfiles/tmpfiles.c

```diff
@@ -110,6 +110,14 @@ typedef enum ItemType {
         ADJUST_MODE = 'm', /* legacy, 'z' is identical to this */
 } ItemType;
 
+typedef enum AgeBy {
+        AGE_BY_UNSET = 0,
+        AGE_BY_ATIME = 1 << 0,
+        AGE_BY_BTIME = 1 << 1,
+        AGE_BY_CTIME = 1 << 2,
+        AGE_BY_MTIME = 1 << 3,
```

Comment:
so is this a bit mask? or a regular enum? you seem to implement it mostly as regular one, but here you declare it as a mask?

Reply:
Yes, it is an `enum`, but the shifts were added to see where the `1` would be set (like a flag). Are you suggesting that I change it `1, 2, 4, ...`?

Reply:
no, the `1 << 0`, `1 << 1`, … is the right way to denote bit masks. I just found it confusing that you only allowed one of them to be specifying per tmpfiles.d line, even though it actually was a bitmask already, which suggests you allow combinations. And you *should* allow combinations

Reply:
Oh, OK. Right, now that we want to allow multiple values to be set, we can keep it that way. :)